proposals/2019/Enable Multiple Reports.md

```diff
@@ -0,0 +1,17 @@
+# Enable Multiple Reports in a Single BuildingSync XML Data
+
+## Overview
+At present, BuildingSync allows only a single report in valid XML data. This proposal will expand the schema to allow for multiple reports and will make the necessary additions to the Report type to facilitate this change. The changes are breaking.
+
+## Justification
+A single report per file limits the extent to which reports can be associated with different parts of buildings or systems. This is particularly an issue when data has been collected for part of a building, but the report implicitly applies to all buildings in the file. Furthermore, BuildingSync should be able to track changes over time, and allowing multiple reports means that this will be easier to do. Users can simply add a new report if another audit is done rather than add to previous data. While this isn't strictly necessary, it is a slightly more natural representation and should be easier on users.
+
+## Implementation
+Four steps will be taken to make this change:
+
+  1. The report element will be changed into a containerized list: `<Reports><Report/></Reports>`.
+  2. An ID attribute will be added to the `Report` type. This attribute will not be used elsewhere in the schema yet, but it is good practice to have identifiers available.
```

Comment:
This has been done in this PR https://github.com/BuildingSync/schema/pull/80

Reply:
Oops, forgot about that. Will edit this a bit.